backend/services/implementations/testService.ts

```diff
@@ -0,0 +1,32 @@
+import MgTest, { Test } from "../../models/test.model";
+import { CreateTestRequestDTO, SchoolTestDTO } from "../../types";
+import { getErrorMessage } from "../../utilities/errorUtils";
+import logger from "../../utilities/logger";
+import { ITestService } from "../interfaces/testService";
+
+const Logger = logger(__filename);
+
+class TestService implements ITestService {
+  /* eslint-disable class-methods-use-this */
+  async createTest(test: CreateTestRequestDTO): Promise<SchoolTestDTO> {
+    let newTest: Test;
+
+    try {
+      newTest = await MgTest.create({ ...test });
```

Comment:
```suggestion
      newTest = await MgTest.create(test);
```
i think that should work? dk what the ... is for